packages/boot/test/bootstrapTests/liquidation.ts

```diff
@@ -371,6 +378,23 @@ export const addSTARsCollateral = async t => {
   t.log('add-STARS proposal executed');
 };
 
-export type LiquidationTestContext = Awaited<
-  ReturnType<typeof makeLiquidationTestContext>
->;
+export const ensureVaultCollateral = async (
+  collateralBrandKey: string,
+  t: ExecutionContext<LiquidationTestContext>,
+) => {
+  // TODO: we'd like to have this work on any brand
```

Comment:
nice solution! creates the right abstraction and we can improve its robustness later